x-pack/plugins/apm/server/lib/transaction_groups/fetcher.ts

```diff
@@ -53,7 +53,7 @@ export function transactionGroupsFetcher(
           terms: {
             field: `${TRANSACTION_NAME}.keyword`,
             order: { sum: 'desc' },
-            size: 100
+            size: config.get<number>('apm_oss.topTransactionGroupsSize') || 100
```

Comment:
You need to also add `apm_oss.topTransactionGroupsSize` to the `apm_oss` plugin:

https://github.com/ogupte/kibana/blob/fe02298c85e1d92fa8e4bcbc5b5b6218c0cc0b98/src/legacy/core_plugins/apm_oss/index.js#L9

If you enable the option as of now you should get this error:
```
 FATAL  ValidationError: child "apm_oss" fails because ["topTransactionGroupsSize" is not allowed]
```